.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  extends: '@ethereumjs/eslint-config-defaults',
+  parserOptions: {
+    project: ['./tsconfig.json']
+  },
+  rules: {
+    '@typescript-eslint/no-floating-promises': 'off',
```

Comment:
There seems to be some floating promises that I didn't have the time to resolve yet, I didn't want to make too broad changes from sync to async funcs. The primary problem for this seems to be the addPeer promise in a few places.